assets/src/components/stopCard.tsx

```diff
@@ -91,4 +108,16 @@ const StopCard = ({
   )
 }
 
+export const MapSafeArea = createContext<LeafletPaddingOptions>({})
+
+/**
+ * A <{@link StopCard}/> which provides it's `autoPanPadding` parameter via the
+ * nearest {@link MapSafeArea} Context
+ */
+export const SafeAreaContextStopCard = (props: StopCardProps) => {
+  const safeArea = useContext(MapSafeArea)
+
+  return <StopCard {...props} autoPanPadding={safeArea} />
+}
```

Comment:
I also considered doing 
> `<StopCardWithSafeArea/>`
> and `<StopCard.WithSafeArea/>`

I liked the dot notion for accessing a specialized version of `<StopCard/>` but we don't have any existing components in skate which utilize this to use as a reference for naming convention/style.

Reply:
I like either of the two alternatives you present here - should there be other SafeAreaContext aware components, it seems nice for scanability to have the "important"/differentiating part of the name first!

Reply:
I decided to go with the dot notation! [mbta/skate@`b6869c9` (#1916)](https://github.com/mbta/skate/pull/1916/commits/b6869c9bfee5c1e93e158649775b9a9157560a79)